Integrations/python/test/testKafkaTools.py

```diff
@@ -29,6 +30,15 @@ class TestTableTools(unittest.TestCase):
     def setUpClass(self):
         kt._defineSymbols()
```

Comment:
this is redundant, in fact, it reduces the coverage because 1) this is not how a user would use the module, 2) we lose a chance to test if _defineSymbols() get called automatically behind the scene

Reply:
Good points.  Solid.